apps/cli/src/bin.ts

```diff
@@ -102,37 +102,69 @@ program
   .description("Generate TypeScript types from XState machines")
   .argument("<files>", "The files to target, expressed as a glob pattern")
   .option("-w, --watch", "Run the typegen in watch mode")
-  .action(async (filesPattern: string, opts: { watch?: boolean }) => {
-    if (opts.watch) {
-      // TODO: implement per path queuing to avoid tasks related to the same file from overlapping their execution
-      const processFile = (path: string) => {
-        if (path.endsWith(".typegen.ts")) {
-          return;
-        }
-        writeToFiles([path]).catch(() => {});
-      };
-      // TODO: handle removals
-      watch(filesPattern, { awaitWriteFinish: true })
-        .on("add", processFile)
-        .on("change", processFile);
-    } else {
-      const tasks: Array<Promise<void>> = [];
-      // TODO: could this cleanup outdated typegen files?
-      watch(filesPattern, { persistent: false })
-        .on("add", (path) => {
+  .option(
+    "-p, --poll",
```

Comment:
are those `-p` and `-i` shortcuts matching the Tailwind CLI? Personally, I would not introduce those as we might add some other options in the future and they might be better suited for having those shortcuts